### PR TITLE
[WIP] postgres fixing

### PR DESCRIPTION
**Pull request by jpds**
Hello,

I've now tried running this properly in my testing airflow environment and I noticed that pq uses a different URL syntax to the mysql driver. This fix is incorporated here.

However, I'm getting this error with postgres with `getEventTotalData`'s function call:

```
airflow=# SELECT COUNT(*), COALESCE(dag_id, ''), COALESCE(task_id, ''), event, MAX(id) FROM log WHERE id > ? GROUP BY dag_id, task_id, event;
ERROR:  syntax error at or near "GROUP"
```

It seems to work fine with `id > 0` - could someone help me with this?

### CODE REVIEW

**Comment by DemonTPx** [comment]
I'm just gonna wait a while until you're done with alle these small commits, mate. Let me know when you think it's ready :)

**Comment by jpds** [comment]
@DemonTPx I'm still not sure how to fix the MySQL prepared statement issue, otherwise this would be ready.

**Comment by DemonTPx** [comment]
Could it be possible that `c.lastEventID` fills in the `?` in the query with an empty string instead of a `0`?

**Comment by jpds** [comment]
And this is now ready!

File: collector.go [hunk 1]
```diff
@@ -154,17 +155,34 @@ func getDagData(db *sql.DB) ([]dag, error) {
 	return dagList, nil
 }
 
+func formulateEventTotalDataPreparedStatement() string {
+	databaseBackend := os.Getenv("AIRFLOW_PROMETHEUS_DATABASE_BACKEND")
```
Comment (DemonTPx) [inline collector.go:159]: Resolving stuff from the environment happens (and only should happen) in `main.go`. Also, you can used the `dbDriver` variable for this which has been filled in by `main.go`

**Comment by DemonTPx** [comment]
Thanks :+1:

**Pull request merged**